ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/DebugAccountRange.java

```diff
@@ -54,7 +54,7 @@ public DebugAccountRange(final Supplier<BlockchainQueries> blockchainQueries) {
   public String getName() {
     // TODO(shemnon) 5229b899 is the last stable commit of retesteth, after this they rename the
     //  method to just "debug_accountRange".  Once the tool is stable we will support the new name.
```

Comment:
think you can remove the TODO also - reading this, that's probably why this one was implemented differently - used to have a different name.